zoo/base/apps.py

```diff
@@ -24,7 +24,7 @@ def ready(self):
 
         celery_app.add_periodic_task(timedelta(hours=1), repos_tasks.sync_repos)
         celery_app.add_periodic_task(timedelta(hours=1), repos_tasks.schedule_pulls)
-        celery_app.add_periodic_task(timedelta(days=1), repos_tasks.sync_zoo_file)
+        # celery_app.add_periodic_task(timedelta(days=1), repos_tasks.sync_zoo_file)
```

Comment:
delete please, we can always get it back from git, no need to have it commented out

Reply:
but... don't we want to have a task for syncing entities? 🤔